src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -126,4 +128,12 @@ public void dateFunctionNameCaseInsensitiveTest() {
                         "ORDER BY date_format(insert_time, 'yyyy-MM-dd', 'UTC')", "jdbc")
         );
     }
+
+    @Test
```

Comment:
Could you add ORDER BY and GROUP BY tests for JDBC format?

Reply:
Yes sure, I will add group by aggregation here. Thanks! @davidcui-amzn

Reply:
I've added the IP type as VARCHAR of JDBC type. And it passed the sanity test in DbVisualizer. Thanks!